spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -312,7 +315,7 @@ The JSON node is named `<metric-name>`. The JSON leafs are named `<key>[';'<tag-
 {
   "responseTime": {
     "count": 29382,
-    "elapsedTime": 25608694,
+    "sum": 25608694,
```

Comment:
I'd suggest that this also stay as elapsedTime for consistency with simpleTimer -- see the conversation from https://github.com/eclipse/microprofile-metrics/pull/621#issuecomment-719547650